lib/dish.rb

```diff
@@ -0,0 +1,14 @@
+class Dish
```

Comment:
More a thought on naming classes... would this be better named as Menu? Feels like the function is about displaying several dishes, not just one dish

Reply:
I agree with Will's feedback here! I would think of this object as more of a 'Menu' as its responsibility is handling multiple Dish items.